test/fixtures/input/app/router.js

```diff
@@ -1,11 +1,13 @@
+import classic from 'ember-classic-decorator';
 import EmberRouter from '@ember/routing/router';
 import config from './config/environment';
 
-const Router = EmberRouter.extend({
-  location: config.locationType,
-  rootURL: config.rootURL,
-});
+@classic
+class RouterRouter extends EmberRouter {
```

Comment:
Same here, RouterRouter doesn't sound nice to me.

Reply:
reverted